### DISCUSSION TITLE
Processing .wiff on Linux

### DISCUSSION THREAD

**Discussion by hadrianprey** [vdemichev/DiaNN#988] [Q&A] [answered]
I'm running DIA-NN on a Linux distribution; however, I'm encountering issues when using .wiff files. I've tried installing ProteoWizard (I've previously tested .wiff files on Windows without any problems) but it still doesn't recognize the files and fails. Is it possible to use these .wiff files on Linux?

**Comment by andrewlai** [comment, marked as answer]
To read .wiff files, you will need the specific .dll files from Sciex, which I believe are Windows-only. I think for Linux, you would need to convert the files into another format first, such as .mzML or .dia

**Reply by hadrianprey** [reply]
Yes, that's what i was thinking, but i wanted to be shure about it. I already did the conversion to mzML (using proteo wizard btw), the main problem is that the size of the file grows a lot, even using the optimization options of proteo wizard, for example a 12MB file grows up to 3 GB, i don't know if you have any info about that? Anyway thanks a lot for the answer!

**Reply by vdemichev** [reply]
What's the size of .wiff.scan? In general, please use the recommended conversion settings (32-bit, peak picking), this will reduce the file size. It will still be larger than .wiff.scan (because Sciex uses very good compression), but quite manageable. Or can convert to .dia under Windows.

**Reply by hadrianprey** [reply]
The size is about 600MB, anyway I will try converting to .dia or using Wine instead.
Thank you so much!
